src/js/main.js

```diff
@@ -0,0 +1,225 @@
+let todoList= [];
+let storage = localStorage;
+let count=0;
+
+class Todo{
+  constructor(id, name, completed)
+  {
+    this.id = id+"";
+    this.name = name;
+    this.completed = completed;
+  }
+}
+
+window.onload= function(){
+    if(storage.getItem("list")===null) return;
+
+    const storageTodo = storage.getItem("list");
```

Comment:
storage key는 상수로 선언해서 쓰면 중복 하드 코딩도 방지하고 가독성도 더 좋아질 것 같아요 :)
```suggestion
    const STORAGE_KEY_TODO_LIST = "list";
    const storageTodo = storage.getItem(STORAGE_KEY_TODO_LIST);
```